capstone/src/main/webapp/script.js

```diff
@@ -122,10 +134,12 @@ function precinctControl(controlDiv, map) {
   //* *button functionality */
   dataUI.addEventListener('click', function() {
     precinctButtonOn = !precinctButtonOn;
-    if (precinctButtonOn) {
+    if (!precinctButtonOn) {
```

Comment:
Thank you for adding the boolean here, it works well with the visibility property